pkgs/mcaptcha-cache/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+}: let
+  src = fetchFromGitHub {
+    owner = "mCaptcha";
+    repo = "cache";
+    rev = "67d6c701baa804849abc53a78422a6da01358487";
+    # NOTE: Avoiding this typo fix (which caused a bug in libmcaptcha)
+    # https://github.com/mCaptcha/cache/commit/f30bc54e6374cf5fad07af8f3d38bbe5fbbb4b20
+    # until this is merged https://github.com/mCaptcha/libmcaptcha/pull/12
+    sha256 = "sha256-whRLgYkoBoVQiZwrmwBwqgHzPqqXC6g3na3YrH4/xVo=";
+  };
+in
+  rustPlatform.buildRustPackage rec {
+    inherit src;
+    pname = "cache";
+    version = "unstable-2023-03-08";
+
+    cargoLock = {
+      lockFile = ./Cargo.lock;
+      outputHashes = {
+        "libmcaptcha-0.1.4" = "sha256-KwFT0Px5ZQGa26fjkiaT8lKc8ASVdfL/67E0hnaHl7I=";
+      };
+    };
+
+    nativeBuildInputs = [rustPlatform.bindgenHook];
+
+    # We are unable to figure out the following error 🤷
+    #
+    # ```
+    # warning: `mcaptcha-cache` (lib test) generated 4 warnings
+    #   Finished test [unoptimized + debuginfo] target(s) in 0.03s
+    #   Running unittests src/lib.rs (target/debug/deps/cache-e9c2ad24991bfc21)
+    # thread panicked while processing panic. aborting.
+    # error: test failed, to rerun pass `--lib`
+    #
+    # Caused by:
+    # process didn't exit successfully: `<redacted>/mcaptcha-cache/target/debug/deps/cache-e9c2ad24991bfc21` (signal: 6, SIGABRT: process abort signal)
+    # ```
+    doCheck = false;
```

Comment:
Advice from @realaravinth: use `make test` instead (which uses `cargo test  --all --all-features --no-fail-fast`, which doesn't crash in this way).

Reply:
Done! (we skipped using the makefile because it does some stuff with venvs that doesn't immediately play nicely with nix)